repository-data/application/src/main/resources/hcm-config/configuration/queries/templates/new-banner-document.yaml

```diff
@@ -0,0 +1,25 @@
+---
+definitions:
+  config:
+    /hippo:configuration/hippo:queries/hippo:templates/new-banner-document:
```

Comment:
Where is this used?

Reply:
https://www.onehippo.org/library/concepts/document-types/template-queries.html

Reply:
I get what it is but it's not used anywhere currently right? See comment below in banners.yaml, I think it should be used there?